src/app/footer/footer.component.ts

```diff
@@ -59,9 +48,17 @@ export class FooterComponent implements OnInit {
     return about.name;
   }
 
-  constructor() { }
+  constructor(private categoriesList: FooterService) { }
 
   ngOnInit(): void {
+    this.getFooterCategories();
```

Comment:
rename func to getAllCategories or fetchCategories

Reply:
@akashsinha13  changed to getAllCategories